src/cursor.c

```diff
@@ -446,6 +432,11 @@ process_cursor_motion(struct server *server, uint32_t time)
 		return;
 	}
 
+	if (seat->drag.active) {
+		wlr_scene_node_set_position(&seat->drag.icons->node,
+			seat->cursor->x, seat->cursor->y);
```

Comment:
After some comments from @johanmalm via IRC I moved everything DnD related to `src/dnd.c` so we don't collect random stuff in `src/cursor.c`. This part is directly accessing a `src/dnd.c` "owned" scene node to update its position.

Should I add another wrapper for this in `src/dnd.c` instead? Something like on of these?
```c
dnd_icons_update(struct seat *seat);
dnd_icons_move(struct seat *seat);
dnd_icons_reposition(struct seat *seat);

```

Reply:
After further discussion via IRC it was decided to use
```c
dnd_icons_move(struct seat *seat, double x, double y);
```